services/app/.env.example

```
@@ -0,0 +1 @@
+DATABASE_URL="postgresql://root@localhost:26257/notifications?sslmode=disable"
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)